aether-kernel/aether/kernel/api/tests/__init__.py

```diff
@@ -78,6 +90,34 @@
     ],
 }
 
+EXAMPLE_NESTED_SCHEMA = {
+  'type': 'record',
+  'fields': [
+    {
+      'name': 'name',
+      'type': 'string'
+    },
+    {
+      'name': 'location',
+      'type': {
+        'type': 'record',
+        'fields': [
+          {
+            'name': 'lat',
+            'type': 'int'
+          },
+          {
+            'name': 'lng',
+            'type': 'int'
+          }
+        ],
+        'name': 'Nested'
+      }
+    }
+  ],
+  'name': 'Nested'
+}
```

Comment:
In python the indentation is with 4 spaces ;-)

Reply:
Where is my linter when I need it!?